app/src/main/java/database/DatabaseHelper.kt

```diff
@@ -0,0 +1,25 @@
+package database
+
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+
+class DatabaseHelper(context: Context) : SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {
```

Comment:
come proposta direi che potremmo fare una sola tabella `misurazioni/measurements` con le colonne `valore`, `classificazione` e `monitor` anziché gestire le tabelle separatamente;
aggiungiamo un metodo privato `storeMeasurement` che prende il valore, la classificazione e il (tipo di) monitor che ha fatto la misurazione;
in ultimo aggiungiamo i metodi pubblici tipo `storeAudioMeasurement` che si occupano di chiamare `storeMeasurement` specificando solo il monitor